nilearn/datasets/utils.py

```diff
@@ -154,12 +159,17 @@ def _chunk_read_(response, local_file, chunk_size=8192, report_hook=None,
     bytes_so_far = initial_size
 
     t0 = time.time()
+    _chunk_period = total_size // (chunk_size * report_frequency)
```

Comment:
If total_size is None, the whole remaining content is downloaded. So normally it's not None at this point.
see https://github.com/nilearn/nilearn/pull/976/files#diff-c22120ae66106d07e452aed0f968c08eR150